apps/docs/stories/components/BankAccountForm/index.stories.tsx

```diff
@@ -0,0 +1,155 @@
+import type { Meta } from "@storybook/web-components";
+import { withActions } from "@storybook/addon-actions/decorator";
+import {
+  CSSVarsExample,
+  paymentMethodFormComponentMethods,
+  StoryBaseArgs,
+} from "../../utils";
+
+import "@justifi/webcomponents/dist/module/justifi-bank-account-form";
+
+const storyBaseArgs = new StoryBaseArgs([
+  "account-id",
+  "client-id",
+  "iframe-origin",
+  "theme",
+]);
+storyBaseArgs.argTypes["client-id"].table.disable = true;
+storyBaseArgs.argTypes["account-id"].table.disable = true;
```

Comment:
Why are these being hidden? If it's because we only want to show how to use the component with web component tokens, should that be added in as an arg and displayed on the table?

Reply:
These ones as well were not modified by this PR. I am not sure why they are being hidden. We can change that if we want it makes sense.